PayPalWebCheckout/src/main/java/com/paypal/android/checkoutweb/PayPalWebCheckoutFunding.kt

```diff
@@ -0,0 +1,8 @@
+package com.paypal.android.checkoutweb
+
+/**
+ * Enum class to specify the type of funding for an order
+ */
+enum class PayPalWebCheckoutFunding(val value: String) {
```

Comment:
Wondering about naming on this - would `PayPalWebCheckoutFundingSource` or `PayPalWebCheckoutFundingType` be more clear for this, or are those options too long?

Reply:
it could be `FundingSource`. As im not a native english speaker, does the concept of "order's funding" make sense? My point being that if adding `Source` or `Type` adds more info to the merchant.

Also, i think we need to maybe re visit the naming convention of adding the module name as a prefix, because class name start to get really long :(

Reply:
I think `PayPalWebCheckoutFundingSource` makes the most sense here and lines up with what we call a funding source in other SDKs (like NXO). The naming is quite long but I think we are going to want to lean towards more verbose naming to avoid naming collisions with NXO.